### PR TITLE
Make "Activity List" and "Popular Adventures" blocks responsive

### PR DESCRIPTION
**Pull request by kienstra**
Hi @miina,
Could you please review this PR, which makes 2 blocks responsive:

activity-list-mobile

popular-adventures-mobile

It also applies styling to the editor. The "Activity List" still shows in 2 rows if the panel is present and the screen is the size of a MacBook. But it looks good if the panel isn't present.

activity-list

### CODE REVIEW

File: assets/css/editor-blocks.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
...:hover{-webkit-transform:scale(1.02) translateY(-2%);transform:scale(1.02) translateY(-2%);box-shadow:0 .4rem 2rem rgba(50,58,67,.4)}.h1.line-height-2,.h2.line-height-2,.h3.line-height-2,.h4.line-height-2,.h5.line-height-2,.h6.line-height-2,h1.line-height-2,h2.line-height-2,h3.line-height-2,h4.line-height-2,h5.line-height-2,h6.line-height-2{line-height:1.4}.ampstart-btn{-webkit-transition:box-shadow .25s,-webkit-transform .25s;transition:box-shadow .25s,-webkit-transform .25s;transition:box-shadow .25s,transform .25s;transition:box-shadow .25s,transform .25s,-webkit-transform .25s;line-height:1.125rem;padding:.5rem 1rem;border-color:transparent}.ampstart-btn[disabled]{opacity:.5;cursor:default}.ampstart-btn:not([disabled]):hover{-webkit-transform:scale(1.02) translateY(-2%);transform:scale(1.02) translateY(-2%);box-shadow:0 .4rem 2rem rgba(50,58,67,.4)}.ampstart-btn:not([disabled]):focus{outline:none;box-shadow:none}.ampstart-btn.bold{font-weight:600}.travel-date-input{line-height:1.125rem;padding:.5rem 1rem;padding-left:calc(.5rem * 2.5 + 1.2em)}.travel-date-input .travel-icon{position:absolute;left:0;top:0;box-sizing:content-box;padding:.5rem 1rem}.travel-date-input.input-dark .t
```
Comment (kienstra) [inline assets/css/editor-blocks.css]: The diff here is much simpler than it looks. I simply removed the style rule `min-width:100vw` from `.travel-overflow-container`.

File: blocks/activity-list/index.js [hunk 2]
```diff
@@ -58,23 +58,21 @@ export default registerBlockType(
 					<div className='max-width-3 mx-auto px1 md-px2'>
 						<h3 className='bold h1 line-height-2'>{ heading }</h3>
 					</div>
-					<div className='overflow-scroll'>
```
Comment (kienstra) [inline blocks/activity-list/index.js:61]: This diff also looks more extensive than it is. I simply deleted the line above and its closing `</div>`, and re-indented its child elements.

File: includes/class-amp-travel-blocks.php [hunk 3]
```diff
@@ -277,7 +277,7 @@ public function render_block_activity_list( $attributes ) {
 					</div>';
 		}
 		$output .= "<div class='travel-overflow-container'>
-							<div class='flex justify-center p1 md-px1 mxn1'>";
+							<div class='flex flex-wrap flex-auto justify-center p1 md-px1 mxn1'>";
```
Comment (kienstra) [inline includes/class-amp-travel-blocks.php:280]: These classes allow the circles of activities to wrap to the next line if they fill the line.

**Review by miina — APPROVED** [review]
@kienstra The changes look good, thanks for fixing this.

**Pull request merged**